source/ProgArgs.cpp

```diff
@@ -2989,6 +3011,7 @@ void ProgArgs::setFromPropertyTreeForService(bpt::ptree& tree)
 	runS3ListObjNum = tree.get<uint64_t>(ARG_S3LISTOBJ_LONG);
 	runS3ListObjParallel = tree.get<bool>(ARG_S3LISTOBJPARALLEL_LONG);
 	runS3MultiDelObjNum = tree.get<uint64_t>(ARG_S3MULTIDELETE_LONG);
+	runS3StatDirs = tree.get<uint64_t>(ARG_S3STATDIRS_LONG);
```

Comment:
wrong type in `tree.get<uint64_t>`. should be `<bool>`.

Reply:
all 5 new options here also need to appear in `ProgArgs::getAsPropertyTreeForService`, which is the counter-part to this function to transfer option values to the "elbencho --service" instances.